src/main/kotlin/no/nav/dokgen/configuration/SwaggerConfig.kt

```diff
@@ -1,25 +1,21 @@
 package no.nav.dokgen.configuration
 
+import org.springdoc.core.GroupedOpenApi
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Primary
 import org.springframework.hateoas.client.LinkDiscoverers
 import org.springframework.hateoas.mediatype.collectionjson.CollectionJsonLinkDiscoverer
 import org.springframework.plugin.core.SimplePluginRegistry
-import springfox.documentation.builders.PathSelectors
-import springfox.documentation.builders.RequestHandlerSelectors
-import springfox.documentation.spi.DocumentationType
-import springfox.documentation.spring.web.plugins.Docket
 
 
 @Configuration
 class SwaggerConfig {
     @Bean
-    fun api(): Docket {
-        return Docket(DocumentationType.SWAGGER_2)
-            .select()
-            .apis(RequestHandlerSelectors.any())
-            .paths(PathSelectors.any())
+    open fun publicApi(): GroupedOpenApi? {
+        return GroupedOpenApi.builder()
+            .group("springshop-public")
+            .pathsToMatch("/public/**")
```

Comment:
Skal den bare gjelde for API som er på path .../public/** ?
Fra tidligere så er den aktivert for "any" -> .paths(PathSelectors.any())

Reply:
Nei, fjernet slik at det gjelder for any som før